etl/steps/data/meadow/terrorism/2023-07-20/global_terrorism_database.py

```diff
@@ -0,0 +1,81 @@
+"""Load a snapshot and create a meadow dataset."""
+
+from typing import cast
+
+import pandas as pd
+from owid.catalog import Table
+
+from etl.helpers import PathFinder, create_dataset
+from etl.snapshot import Snapshot
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+    """
+    Process global terrorism data from the years 2020 and 2021 and save the results in a new Meadow dataset.
+
+    Args:
+        dest_dir (str): The directory where the new Meadow dataset will be created.
+
+    targtype1_txt: the target/victim type field captures the general type of target/victim.
+    weaptype1_txt:  the type of weapon used in the incident.
+    attacktype1_txt:  the hierarchy of attack types, including:
+          - Assassination
+          - Hijacking
+          - Kidnapping
+          - Barricade Incident
+          - Bombing/Explosion
+          - Armed Assault
+          - Unarmed Assault
+          - Facility/Infrastructure Attack
+          - Unknown
+
+
+    Casualty Fields:
+        - nkill: total confirmed fatalities for the incident
+        - nwound: confirmed non-fatal injuries to both perpetrators and victims
+    """
+    #
+    # Load inputs.
+
+    #
+    # Retrieve snapshots for terrorism data up intil 2020 and terrorism data between 2020-2021.
+    snap_2020 = cast(Snapshot, paths.load_dependency("global_terrorism_database.xlsx"))
+    snap_2021 = cast(Snapshot, paths.load_dependency("global_terrorism_database_2021.xlsx"))
+    # Select columns of interest
+    COLUMNS_OF_INTEREST = [
+        "iyear",
+        "country_txt",
+        "region_txt",
+        "attacktype1_txt",
+        "weaptype1_txt",
+        "targtype1_txt",
+        "nkill",
+        "nwound",
+    ]
+
+    # Load data from snapshots.
+    df_2020 = pd.read_excel(snap_2020.path)
```

Comment:
Loading the excel file took almost 5 minutes on my laptop. It might be better to convert it to `.csv` format on your local and then upload the csv as a snapshot instead of excel file (it might not be crystal "raw", but it looks like there's nothing extra in the excel file).

(For comparison, csv file is 5x smaller and loads in 0.3s. Using `feather` format would compress even better, but it's not as common as csv)